### DISCUSSION TITLE
Can I set table field comment for MySQL?

### DISCUSSION THREAD

**Discussion by oliverzy** [prisma/prisma#4800] [Q&A] [answered]
I searched the doc of Prisma. But I cannot find anything.

**Comment by nikolasburk** [comment]
Hey @oliverzy 👋 Can you elaborate a bit on your question and maybe give an example?

You can write comments in the Prisma schema using `//` and `///`, you can find the documentation for that [here](https://www.prisma.io/docs/concepts/components/prisma-schema#comments).

Please let me know if that helps or if you have any further questions, always happy to help 🙌

**Reply by oliverzy** [reply]
in mysql ddl, you can add comment for each table field so that you can understand what the field means in the database tools. prisma comment only exists in prisma schema. It looks like prisma migrate will not put it into ddl. That’s what I means, thanks

**Reply by Andrewzzz** [reply]
Same question here. I cannot use migrate to add comments to tables or fields in database (expecting something like "COMMENT ON" in PostgreSQL DDL).

**Reply by janpio** [reply, marked as answer]
Right now this is not supported indeed.

You _can_ use `--create-only` for Migrate, and then modify the generated SQL to add the comments you require.

**Reply by hiphapis** [reply]
@janpio Don't you have any plan for this? If i make PR for this, is merge-able PR?
cuz i need this feature

**Reply by janpio** [reply]
There are multiple issues for support of this here: https://github.com/prisma/prisma/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+comments+label%3A%22topic%3A+comments%22 A PR for this is probably not possible, as we will need to define how to define these comments in our Prisma Schema Language, and also how to reconciliate this with our existing comments with `//` and `///`.

**Reply by luatnd** [reply]
Yes this is very useful, my db was lost all the field comment after using prisma :D
At least, supporting /[a-z 0-9]/ string is very appriciated.

**Reply by melroy89** [reply]
Still not solved in 2026

**Comment by ekil1100** [comment]
@hiphapis @janpio Any update and progress on this feature?

**Reply by janpio** [reply]
No, and there will not be here in this discussion. Follow the feature requests issues please: https://github.com/prisma/prisma/discussions/4800#discussioncomment-1447683

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!

**Comment by Azmeer-crescent** [comment]
For inspiration, here's how Laravel Migration does it:

```
Schema::create('patients', function (Blueprint $table) {
    $table->id();
    $table->string('email')->comment('Patient email address');
    $table->timestamps();
});
```

**Reply by melroy89** [reply]
It's crazy that Prisma still doesn't have this basic support for comments in columns and tables (stored in mysql/mariadb).
